app/src/main/java/com/netguru/codereview/network/ShopListApi.kt

```diff
@@ -0,0 +1,11 @@
+package com.netguru.codereview.network
+
+import com.netguru.codereview.network.model.ShopListItemResponse
+import com.netguru.codereview.network.model.ShopListResponse
+import kotlinx.coroutines.flow.Flow
+
+interface ShopListApi {
+    suspend fun getShopLists(): List<ShopListResponse>
+    suspend fun getShopListItems(listId: String): List<ShopListItemResponse>
+    fun getUpdateEvents(): Flow<String>
```

Comment:
What do `getUpdateEvents` do? Why it is not suspend?
For other methods:
Please, add POST/GET annotation. Also, if it is Refrofit, I do not see OhHttp configuration